modules/ztp-image-based-upgrade-with-backup.adoc

```diff
@@ -0,0 +1,229 @@
+
+When you have generated the seed image and have completed the `Prep` stage, you can begin upgrading the target clusters.
+During the upgrade process, the OADP Operator creates a backup of the cluster, then {lcao} reimages it.
+
+[IMPORTANT]
+====
+If the upgrade fails or stops, you must roll back the upgrade.
+For more information, see "(Optional) Initiating rollback of the {sno} clusters after an image-based upgrade".
+
+In case of a rollback, all the resources created by the {lcao} are automatically deleted.
+====
+
+.Prerequisites
+
+* You have completed the `Prep` stage.
+* You have installed the OADP Operator.
+* You can an Object storage with your configured credentials.
+// Install OADP and storage configgged: Is it okay to have it in the prereqs? Or is this something that the user would expect to be guided through? With the different storage solutions, we probably can't cover everything.
+
+.Procedure
+
+This example demonstrates how to back up and upgrade a cluster with applications on a persistent volume.
+
+[NOTE]
+====
+The target cluster does not need to be detached from the hub cluster.
+====
+
+. Create the secret with your credentials to your Object storage.
+. Create the configuration for the OADP Operator to access your Object storage instance.
+//TODO Check existing OADP docs for steps, link to them if applicable
+
+. Specify what you want to back up:
+//TODO decide how to present this info. Might be best to create a reference section with all the example CRs.
+
+`backup_acm_klusterlet.yaml`:: This allows you to reattach the cluster to the hub cluster once the upgrade and restore are completed.
+`backup_localvolume.yaml`:: This allows to restore the persistent volume that contains the `index.html` of your application.
+`backup_app.yaml`:: The information that must be backed up is different for each application. This CR specifies what needs to be backed up to recover the applications.
+
+. Define the restore order for the OADP Operator in the `Restore` CR by using the `lca.openshift.io/apply-wave` field:
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: acm-klusterlet
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "1"
+spec:
+  backupName:
+    acm-klusterlet
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: apache-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
+spec:
+  backupName:
+    apache-app
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: localvolume
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2"
+spec:
+  backupName:
+    localvolume
+----
+
+. Create a `ConfigMap` CR for the backup and restore information that the {lcao} uses during the upgrade.
```

Comment:
I think this is important to have some context. So, this configMap basically contains CRs that are understood by OADP so it can configure the backup & restore. Since LCA is in charge of orchestrating the whole backup&restore process, it gets this info from a configmap and passes it to OADP at the right moment of the upgrade process.  That's why we need to generate a configMap from OADP CRs.